rtgui/guiutils.cc

```diff
@@ -1010,13 +1028,31 @@ bool MyComboBox::on_scroll_event (GdkEventScroll* event)
     return false;
 }
 
+void MyComboBox::setPreferredWidth (int minimum_width, int natural_width)
+{
+    if (natural_width==-1 && minimum_width==-1) {
+        naturalWidth = minimumWidth = 70;
+    } else if (natural_width==-1) {
+        naturalWidth =  minimumWidth = minimum_width;
+    } else if (minimum_width==-1) {
+        naturalWidth = natural_width;
+        minimumWidth = rtengine::max(naturalWidth/2, 20);
```

Comment:
What is wrong with `std::max`? Would be one less cross-dependency into `rtengine`.